.github/workflows/stale.yml

```
@@ -0,0 +1,21 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/stale@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'Stale issue message'
+          stale-pr-message: 'This PR has been marked as stale due to no activity for 60 days. We are still experimenting with the new workflow feature but please close the issue yourself & track it through gitlab if you are not actively engaged with it anymore so we can prioritise our reviewer resources. Stale PRs will be autoclosed after 60 days .... we think ... but we are still experimenting & testing this at the moment. See https://github.com/civicrm/civicrm-core/pull/15573 for latest...'
+          stale-issue-label: 'no-issue-activity'
+          stale-pr-label: 'no-pr-activity'
+          days-before-stale: 60
+          days-before-close: 60
```

I commented on the list that it would be good to give a warning after 60 days and then auto-close after 75 days of inactivity, thinking that this both warned and closed at 60 days.  However, looking at the example in https://github.com/actions/stale/blob/master/README.md, it seems that the `days-before-close` is the number of days that a PR or issue sits after being labeled "stale" before it's closed.  I think that means this code as it stands would auto-close after a total of 120 days of inactivity.

```suggestion
          days-before-close: 15
```

I'm suggesting here that it sit labeled "stale" for only 15 extra days before closing.

---

yep - I had it set to not autoclose until 120 days - I'm open to experimenting with variants of time frames

---

I don't have strong opinions, but I think we probably shouldn't have more time after the notification than is sufficient to be sure people have a chance to bump the issue and save it from autoclosing.  Two or three weeks is probably enough--just long enough that if it happens while you're on vacation you'll see it and can comment on it.  By and large, anything still stale after that point is never going to get bumped and is just taking up space.